src/earn/EarnConfirmationScreen.tsx

```diff
@@ -139,28 +146,29 @@ export default function EarnCollectScreen({ route }: Props) {
   return (
     <SafeAreaView style={styles.container}>
       <ScrollView contentContainerStyle={styles.contentContainer}>
-        <Text style={styles.title}>{t('earnFlow.collect.title')}</Text>
+        <Title mode={mode} />
         <View style={styles.collectInfoContainer}>
-          <CollectItem
-            title={t('earnFlow.collect.total')}
-            tokenInfo={depositToken}
-            rewardAmount={withdrawAmountInDepositToken}
-          />
+          {mode !== 'Claim' && (
+            <CollectItem
+              title={t('earnFlow.collect.total')}
+              tokenInfo={depositToken}
+              rewardAmount={withdrawAmountInDepositToken}
+            />
+          )}
           {rewardsTokens.map((token, index) => (
             <CollectItem
-              title={t('earnFlow.collect.plus')}
+              title={t('earnFlow.collect.reward')}
```

Comment:
For partial withdraws we don't claim rewards, right? So should these not be displayed?

Reply:
Fixed in 7c0d8750b4fe1f76c2b9cce3f0e728b5440edb5b & tests updated in b2d24bd521c22266c508e812ada8761392b0160f. I also added a todo to update the tests to ensure the reward positions are not included in the prepared transaction for partial withdrawals as they currently are until #6192 is merged.